index.bs

```diff
@@ -851,20 +851,18 @@ into a different parent element. An example for carrying out such an attack
 is by relying on the change of parsing behavior for foreign content or
 misnested tags.
 
-The Sanitizer API offers help against Mutated XSS, but relies on some amount of
-cooperation by the developers. The `sanitize()` function does not handle strings
-and is therefore unaffected. The `setHTML` function combines sanitization
-with DOM modification and can implicitly apply the correct context. The
-`sanitizeFor()` function combines parsing and sanitization, and relies on the
-developer to supply the correct context for the eventual application of its
-result.
-
-If the data to be sanitized is available as a node tree, we encourage authors
-to use the `sanitize()` function of the API which returns a
-DocumentFragment and avoids risks that come with serialization and additional
-parsing. Directly operating on a fragment after sanitization also comes with a
-performance benefit, as the cost of additional serialization and parsing is
-avoided.
+The Sanitizer API offers only functions that turn a string into a node tree.
+The context is supplied implicitly by all sanitizer functions:
+`Element.setHTML()` uses the current element; `Document.parseHTML()` creates a
+new document. Therefore Sanitizer API is not directly affected by mutated XSS.
+
+If a developer were to retrieve a sanitized node tree as a string, e.g. via
+`.innerHTML`, and to then parse it again then mutated XSS may occur.
+We recommend against this practice. If processing or passing of HTML as a
```

Comment:
Maybe "discourage" instead of "recommend against"? For me, as a non-native speaker, the latter sounds a little off.

Reply:
Done.